src/main/java/org/example/awt/trafficLight.java

```diff
@@ -0,0 +1,136 @@
+import java.awt.*;
+import java.awt.event.*;
+
+
+class lightDrawing extends Component{
```

Comment:
Change class name to LightDrawing to follow CamelCase comvention for classes.

Please refer this https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html